examples/nat_scapy_tests.py

```diff
@@ -144,9 +144,11 @@ def run_server2(serverport=12022, natport=35002):
     ackn += 1
     print "Replying with Ack"
     send(ip/TCP(sport=serverport,dport=natport,flags="A", seq=seqn,ack=ackn))
+    time.sleep(1)
 
     ## Should now be unable to use the connection
     # Send something
+    time.sleep(1)
     seqn += 1
     ackn += 1
```

Comment:
It's a feeble attempt to imitate the correct TCP protocol. As I said in another comment, these tests aren't very robust.